src/task.py

```diff
@@ -66,18 +67,20 @@ def landcover_reclass(self, lc_val, elev_zone, zone):
         )
 
     def calc(self):
+
         lc_height = self.reclass_table.filter(ee.Filter.eq(INCLUDE_HEIGHT, 1))
         lc_height_vals = lc_height.aggregate_array(LC_VALUE_LABEL)
         lc_no_height = self.reclass_table.filter(ee.Filter.eq(INCLUDE_HEIGHT, 0))
         lc_no_height_vals = lc_no_height.aggregate_array(LC_VALUE_LABEL)
 
-        forest_height_mask = (
-            self.forest_height.updateMask(self.watermask)
-            .gte(self.thresholds["forest_height_height_threshold"])
-            .reduceResolution(reducer=ee.Reducer.mean(), maxPixels=125)
-            .reproject(scale=self.scale, crs=self.crs)
-            .gte(self.thresholds["forest_height_cover_threshold"] / 100)
-        )
+        if self.height:
+            forest_height_mask = (
```

Comment:
Hmmm... Not good to put var definition inside a conditional. If `forest_height_mask` is only used inside `str_hab_by_zone`, I'd combine this with that conditional block. If not, I'd restructure so that the minimal variable is defined outside the method, and then overridden if `self.height` exists. Ping me if this isn't clear...